src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -126,6 +126,7 @@ internal ReflectTypeDescriptionProvider()
             [typeof(TimeSpan)] = typeof(TimeSpanConverter),
             [typeof(Guid)] = typeof(GuidConverter),
             [typeof(Uri)] = typeof(UriTypeConverter),
+            [typeof(Version)] = typeof(VersionConverter),
```

Comment:
Sure would be nice if every type listed here had a comment on the class itself or in docs someplace that stated it had a type converter. The first place I looked when serialization broke for me on the upgrade to 3.1 from 2.2 was on the Version class and I didn't see it. I searched for usages on GitHub and it's parser must have problems with `[typeof(Version)]` as I searched for `typeof(Version)` and `typeof(System.Version)`. https://github.com/JamesNK/Newtonsoft.Json/issues/2243